wallet/src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -179,14 +179,14 @@ type CachedCAccModifier = CAccModifier
 -- | Evaluates `txMempoolToModifier` and provides result as a parameter
 -- to given function.
 fixingCachedAccModifier
-    :: (WalletWebMode m, MonadKeySearch key m)
+    :: (WalletWebMode m, MonadKeySearch key m, HasCoreConstants)
```

Comment:
I forgot to remove it here (I started to add `HasCoreConstants` and realised it would be a problem)